### PR TITLE
Using option in console

### PR DESCRIPTION
**Pull request by onon1101**
- fault case:

```txt
➜  Sylinteri git:(cmdline) ✗ ./cmake-build-debug/SYLINTERI -s report-sample/test_report.json -c score-sample/score_sample.json -i          
option needs value: --id
usage: ./cmake-build-debug/SYLINTERI --score=string --config=string --id=string [options] ... 
options:
  -s, --score     score file (string)
  -c, --config    config of score file (string)
  -i, --id        the student id (string)
  -?, --help      print this message

```

- Successful case:
```txt
➜  Sylinteri git:(cmdline) ✗ ./cmake-build-debug/SYLINTERI -s report-sample/test_report.json -c score-sample/score_sample.json -i 111590009
[2024-10-29 22:00:53.737] [info] Got Google report file. [report-sample/test_report.json]
[2024-10-29 22:00:53.740] [info] Got score configuration file. [score-sample/score_sample.json]
[2024-10-29 22:00:53.740] [info] Got student ID: 111590009
[2024-10-29 22:00:53.740] [info] Score configuration: 
[2024-10-29 22:00:53.740] [info]  - TICKET_SYSTEM_TEST -> 100
[2024-10-29 22:00:53.740] [info] Load test suites by name TICKET_SYSTEM_TEST with 11 tests

and so on...

```

### CODE REVIEW

File: src/cmdline.h
Comment (c1ydehhx) [inline src/cmdline.h:1]: I think there may have some cmdtool for it. We don't need to hardcoded it.

Comment (onon1101) [reply src/cmdline.h:1]: It is a effortless way to create a new feature in this case.
The source is here. https://github.com/tanakh/cmdline/blob/master/cmdline.h

Comment (c1ydehhx) [reply src/cmdline.h:1]: How about this?
https://github.com/CLIUtils/CLI11

Comment (onon1101) [reply src/cmdline.h:1]: oh, that cool. It being change my mind.

Comment (onon1101) [reply src/cmdline.h:1]: I will close this pr.

**Pull request closed without merging**